src/main/ui/src/components/Workouts.js

```diff
@@ -159,21 +240,60 @@ export default function Workouts() {
       }
       createExercise(jsonobj)
           .then(response => {
-              console.log("success")
+              getWeights("test@gmail.com")
       });
     };
 
+    const handleAddWorkout = () => {
+      setOpenAddWorkout(false);
+      var daysOfWeek = [];
```

Comment:
We need to have a -1 in this list because empty lists are not supported for dynamo